commands/telepresence/connect.go

```diff
@@ -0,0 +1,75 @@
+package telepresence
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/shipyard/shipyard-cli/pkg/client"
+	"github.com/shipyard/shipyard-cli/pkg/k8s"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var errNoEnvironment = errors.New("environment ID argument not provided")
+
+func NewConnectCmd(c client.Client) *cobra.Command {
```

Comment:
The way this is setup, there is a top-level command "telepresence" that wants the env ID (which it shouldn't). There is a subcommand "connect", which also wants the env ID (which we want). If we want "telepresence" to have subcommands, such as "connect" and any others, there needs to be a dedicated, top-level telepresence command.